http/index.ts

```diff
@@ -130,43 +130,70 @@ async function getRequest<T>(
   options: HttpOptions
 ): AxiosPromise<T> {
   const { params, ...rest } = options;
-  const axiosConfig = addQueryParams(rest, params);
-  const configWithAuth = await withAuth(accountId, axiosConfig);
+  const optionsWithParams = addQueryParams(rest, params);
+  const requestConfig = await withAuth(accountId, optionsWithParams);
+
+  return axios<T>(requestConfig);
+}
+
+async function unauthedGetRequest<T>(options: HttpOptions): AxiosPromise<T> {
+  const { params, ...rest } = options;
+  const optionsWithParams = addQueryParams(rest, params);
+  const requestConfig = await getAxiosConfig(optionsWithParams);
 
-  return axios<T>(configWithAuth);
+  return axios<T>(requestConfig);
 }
 
 async function postRequest<T>(
   accountId: number,
   options: HttpOptions
 ): AxiosPromise<T> {
-  const configWithAuth = await withAuth(accountId, options);
+  const requestConfig = await withAuth(accountId, options);
+  return axios({ ...requestConfig, method: 'post' });
+}
 
-  return axios({ ...configWithAuth, method: 'post' });
+async function unauthedPostRequest<T>(options: HttpOptions): AxiosPromise<T> {
+  const requestConfig = await getAxiosConfig(options);
+  return axios({ ...requestConfig, method: 'post' });
 }
```

Comment:
Is the reason for splitting this into a new method that the callers might now have the `accountId` in context?  If so, what would you think about updating the original methods to take a single object, and make `accountId` optional.  Then if the `accountId` is provided, we generate the auth and make the unauthed request otherwise.

Reply:
Yeah, you don't need an `accountId` to make an unauthed request, so I figured we should assume in those cases that it won't exist. Was trying to approach this in a way that wouldn't make any API changes but I guess since we have the major release going out anyway there's no downside to doing that now. Hmm.

I could go either way. Would definitely clean things up a bit, but I feel like you could also make the case that the vast majority of hubspot endpoints do require auth (and in our case, every single one the CLI interacts with), and moving `accountId` into `options` could obscure the fact that it's necessary for most requests

Reply:
I just saw your comment about Branden's suggestion and I like that as well and it addresses the core of my concern, so I'll flag this resolved.